.github/workflows/lints-stable.yml

```diff
@@ -15,4 +15,4 @@ jobs:
         with:
           name: Clippy (MSRV)
           token: ${{ secrets.GITHUB_TOKEN }}
-          args: --all-features --all-targets -- -D warnings
+          args: --all-features --all-targets -- 
```

Comment:
hm, clippy is not run on the upstream repo?

Reply:
Clippy is run on their CI but there are so few (effectively zero) commits in the recent past that things have broken and nobody has noticed yet.

They aren't immediately fixable either, because the warnings are actually caused by a bug in clippy https://github.com/rust-lang/rust-clippy/issues/12629